src/instructlab/lab.py

```diff
@@ -484,6 +484,16 @@ def generate(
     if endpoint_url:
         api_base = endpoint_url
     else:
+        # Third Party
+        import llama_cpp
+
+        if not llama_cpp.llama_supports_gpu_offload():
+            click.secho(
+                "NOTE: llama_cpp is built without hardware acceleration. "
```

Comment:
```suggestion
                "llama_cpp is built without hardware acceleration. "
```

Is mentioning llama_cpp relevant for the end user?

Reply:
It should be `llama_cpp_python`, not `llama_cpp`. It's relevant because that's what they have to rebuild and reinstall with hardware acceleration.

Reply:
It's just "sad" that users have to have that sort of knowledge, this complicates the user experience. 🤷🏻

Reply:
I hear you and agree with you. #1072 should make it a bit easier on our users.